tests/features/linkedpipes.feature

```diff
@@ -0,0 +1,14 @@
+# linkedpipes.feature
+# Date:  2020.11.02
+# Author: Laurentiu Mandru
+# Email: mclaurentiu79@gmail.com
+
+
+Feature: Test functionality of LinkedPipes ETL running in a Docker container
+
+    Background:
+    Given the baseURI LINKEDPIPES_ETL_URL
+
+  Scenario: Main landing page load scenario
```

Comment:
does this work already? 

Can we have a similar scenario for when the LinkedPipex executor or store is down?